tests/apps/helloworld/service.yaml

```diff
@@ -0,0 +1,42 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: helloworld
+  labels:
+    testapp: helloworld
+spec:
+  selector:
+    testapp: helloworld
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 3000
+  type: LoadBalancer
```

Comment:
Must we rely on external IPs?
This might complicate Minikube/Kind setups..

Also, some cloud providers have limitations on the number of external IPs they can provision.

Reply:
This yaml is just for testing purpose. This will not be used for the actual test because the test util will generate deployment object by itself.

For minikube, it is not difficult to access external ip as long as we get the node ip.

In the first iteration, I would like to use the normal end-to-end case without using port-forward. We can use port-forward later.

Reply:
OK so I'm not sure I understand.. if this YAML won't be used, why is it here?

Reply:
Is this a temporary thing?

Reply:
> Is this a temporary thing?

This is used for testing this helloworld app by using kubectl on your machine without using e2e go test code. 

Let's assume you are creating new test app and go test code for e2e. I believe you want to ensure that new test app is working before writing the go e2e test. in this case, we can deploy helloworld app to your cluster with this yaml.

If you think we do not need this later, we can remove it. this is the initial commit.

Reply:
@yaron2 I added comment in yaml.

Reply:
I will document this process once I complete writing the first e2e test.

Reply:
Also, when it comes to supporting Kind, I am not sure that the general port-forward in client-go is working. Kind provides the different way to map the port - https://kind.sigs.k8s.io/docs/user/quick-start#mapping-ports-to-the-host-machine

We're targeting to Minikube and AKS now, but the test must be platform independent, but some parts depends on the platforms. For example, getting external ip address will depends on the cluster environment. we have to add platform dependent code for this later.